ibm/mas_devops/roles/cluster_monitoring/tasks/update_grafana/main.yml

```diff
@@ -0,0 +1,49 @@
+---
+# 1. Load default storage classes (if not provided by the user)
+# -----------------------------------------------------------------------------
+- name: "update_grafanav4 : Determine storage classes"
```

Comment:
`update_grafanav4` should be `update_grafana` as that is the task name. I think this similar issue is seen in a few other task files where the name doesn't match the task it is now in.

Reply:
Thanks, now changed